pkg/rule/rule.go

```diff
@@ -122,10 +122,17 @@ func (r *Rule) Reason(violation string) string {
 	return reason.String()
 }
 
+func (r *Rule) addNoteToMessage() bool {
+	if r.Options.AddNoteToMessage != nil {
+		return *r.Options.AddNoteToMessage
+	}
+	return false
```

Comment:
Can you add a test case for this scenario?

Reply:
It is covered [here](https://github.com/eedorenko/woke/blob/9d8f8fe94b5b309a4a7af0fc0c7a8126217d36f5/pkg/config/config_test.go#L109) and [here](https://github.com/eedorenko/woke/blob/9d8f8fe94b5b309a4a7af0fc0c7a8126217d36f5/pkg/config/config_test.go#L124)

Reply:
Hmmm... codecov is showing it as uncovered https://app.codecov.io/gh/get-woke/woke/compare/77/diff#diff-cGtnL3J1bGUvcnVsZS5nbw==

Reply:
I guess because it depends on test data in yaml. I'll make a more explicit test, so codecov could understand it.